pkg/ddevapp/traefik_config_middlewares_template.yaml

```diff
@@ -0,0 +1,29 @@
+#ddev-generated
```

Comment:
Please rename this file to show its usage (It's a general *example* file and has nothing really to do with middlewares, right?)

Also, the example file does not seem to have been processed here, and still has go templates in it.